src/views/studio/EditVideoSheet/EditVideoTabsBar/EditVideoTabsBar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { EditVideoSheetTab } from '@/hooks'
+import { IconButton } from '@/shared/components'
+import { SvgGlyphClose, SvgGlyphMinus, SvgGlyphPlus } from '@/shared/icons'
+import { ButtonsContainer, Tab, TabsContainer, TabsHeader, TabTitle, Topbar } from './EditVideoTabsBar.style'
+
+type TabsBarProps = {
+  videoTabs: EditVideoSheetTab[]
+  selectedVideoTab?: EditVideoSheetTab
+  onAddNewTabClick: () => void
+  onRemoveTabClick: (tab: EditVideoSheetTab) => void
+  onTabSelect: (tab: EditVideoSheetTab) => void
+  onCloseClick: () => void
+  onToggleMinimizedClick: () => void
+}
+
+export const EditVideoTabsBar: React.FC<TabsBarProps> = ({
+  videoTabs,
+  selectedVideoTab,
+  onAddNewTabClick,
+  onRemoveTabClick,
+  onTabSelect,
+  onCloseClick,
+  onToggleMinimizedClick,
+}) => (
+  <Topbar>
+    <TabsContainer>
+      <TabsHeader variant="h6">Add new video</TabsHeader>
```

Comment:
I think it should be a clickable button. Also, it's misaligned a bit.
![image](https://user-images.githubusercontent.com/51168865/115351701-d08db600-a1b6-11eb-9dd9-4180f1abe0a4.png)

Reply:
It shouldn't be, it's just a header in the design. Text is aligned properly, it's just that our font will occupy the lower part of its height. Some issue with the font most likely, but that's out of scope here